pages/events.tsx

```diff
@@ -70,33 +70,33 @@ const EventsPage = () => {
       })
   }, [setLoading])
 
+  let futureEvents = events.filter((event) => Date.parse(event.date) > Date.now());
+  let pastEvents = events.filter((event) => Date.parse(event.date) < Date.now());
+
   return (
     <>
       <Masthead title={title} />
       <BlockComponent block={!init}>
         <SectionContainer backgroundColor={theme.palette.background.default}>
-          <Box
-            sx={{
-              display: 'flex',
-              flexDirection: 'column',
-              alignItems: 'center',
-            }}
-            maxWidth={'880px'}
-          >
-            <Stack gap={{ xs: '2.5rem', md: '2rem' }} maxWidth={'880px'}>
-
-              {events.map((event) => (
-                <CardEvent key={event.title} event={event} />
-              ))}
-
-              {events.length === 0 && (
-                <Typography sx={{ textAlign: 'center' }}>
-                  All upcoming events are invite-only. Please check back in the
-                  future for public events.
-                </Typography>
-              )}
-            </Stack>
-          </Box>
+          <Stack gap={{ xs: '2.5rem', md: '2rem' }} maxWidth='880px'>
+            {futureEvents.map((event) => (
+              <CardEvent key={event.title} event={event} />
+            ))}
+            {futureEvents.length === 0 && (
+              <Typography sx={{ textAlign: 'center' }}>
+                All upcoming events are invite-only. Please check back in the
+                future for public events.
+              </Typography>
+            )}
+          </Stack>
+        </SectionContainer>
+        <SectionContainer backgroundColor={theme.palette.background.default}>
```

Comment:
Should we show this section if there are no pastEvents?

Reply:
yeah, if there's no past events, we could just remove the whole 'past events' section entirely.
i was thinking we'd always have past events at this point, but i can still add a conditional here

Reply:
yes, let's remove the section if there is no past events.